tests/gflownet/envs/test_spacegroup.py

```diff
@@ -103,18 +197,17 @@ def test__state2readable2state(env, state):
     )
 
 
-@pytest.mark.skip(reason="Takes considerable time")
+# @pytest.mark.skip(reason="Takes considerable time")
```

Comment:
Exactly, I commented out because it doesn't take too long (compared to other tests I do now. It used to seem relativeluy long at the beginning). I will remove it.